main.go

```diff
@@ -70,30 +72,38 @@ func loggingMiddleware(next http.Handler) http.Handler {
 }
 
 func main() {
-	var config Config
-	err := envconfig.Process("", &config)
-	if err != nil {
-		println("Unable to parse config " + err.Error())
-		os.Exit(1)
-	}
-	rubberneck.Print(config)
-
-	serverRoot, err := fs.Sub(uiFS, "ui")
-	if err != nil {
-		println("Unable to read ui filesystem ", err.Error())
-		os.Exit(1)
-	}
-
-	uiFs := http.FileServer(http.FS(serverRoot))
-	router := mux.NewRouter()
-	router.Use(loggingMiddleware)
+	kong.Parse(&Config)
 
-	router.HandleFunc("/nameifier/{seed}/{count}", nameHandler).Methods("GET")
-	router.HandleFunc("/nameifier/{seed}", blankHandler).Methods("GET")
-	router.HandleFunc("/nameifier/", blankHandler).Methods("GET")
-	router.HandleFunc("/nameifier/", blankHandler).Methods("GET")
-	router.PathPrefix("/").Handler(uiFs)
+	// By default start server
+	if Config.SeedString == "" {
+		rubberneck.Print(Config)
+		serverRoot, err := fs.Sub(uiFS, "ui")
+		if err != nil {
+			println("Unable to read ui filesystem ", err.Error())
+			os.Exit(1)
+		}
 
-	http.Handle("/", router)
-	http.ListenAndServe(fmt.Sprintf("0.0.0.0:%d", config.Port), nil)
+		uiFs := http.FileServer(http.FS(serverRoot))
+		router := mux.NewRouter()
+		router.Use(loggingMiddleware)
+		router.HandleFunc("/nameifier/{seed}/{count}", nameHandler).Methods("GET")
+		router.HandleFunc("/nameifier/{seed}", blankHandler).Methods("GET")
+		router.HandleFunc("/nameifier/", blankHandler).Methods("GET")
+		router.HandleFunc("/nameifier/", blankHandler).Methods("GET")
+		router.PathPrefix("/").Handler(uiFs)
+		http.Handle("/", router)
+		http.ListenAndServe(fmt.Sprintf("0.0.0.0:%d", Config.Port), nil)
+	} else {
+		var allNames []string
+		namer := NewNameifier()
+		for i := 0; i < Config.Count; i++ {
+			name, err := namer.Nameify(fmt.Sprintf("%s-%d", Config.SeedString, i))
+			if err != nil {
+				fmt.Printf("Failed to nameify! %s", err)
+				return
+			}
+			allNames = append(allNames, name)
+		}
```

Comment:
This block is mostly shared with the http service. Let's break it out into a function and call it from both.